zboxcore/sdk/blockdownloadworker.go

```diff
@@ -192,26 +189,29 @@ func (req *BlockDownloadRequest) downloadBlobberBlock() {
 				defer resp.Body.Close()
 			}
 			if resp.StatusCode == http.StatusOK {
-				//req.consensus++
 
-				response, err := ioutil.ReadAll(resp.Body)
-				// if err != nil {
-				// return errors.Wrap(err, fmt.Sprintf("[%d] Read error:\n", req.blobberIdx))
-				// }
+				response, _ := ioutil.ReadAll(resp.Body)
 				var rspData downloadBlock
 				rspData.idx = req.blobberIdx
-				// dec := json.NewDecoder(resp.Body)
-				// err := dec.Decode(&rspData)
 				err = json.Unmarshal(response, &rspData)
+
 				// After getting start of stream JSON message, other message chunks should not be in JSON
 				if err != nil {
 					rspData.Success = true
-					//rawData := make([]byte,0)
-					//json.Unmarshal(response, &rawData)
 					rspData.RawData = response
+
+					// TODO download by 'chunks' should not download full stream and then chunk it
+					// It has to download already chunked data
 					if len(req.encryptedKey) > 0 {
-						// 256 for the additional header bytes,  where chunk_size - 2 * 1024 is the encrypted data size
-						chunks := req.splitData(rspData.RawData, req.chunkSize-2*1024+256)
```

Comment:
why it is 256? encrypt header size is  2*1024 + a 16 gap?

Reply:
@cnlangzi Yes, I am surprised myself too. But this is legacy code that works with auth_tickets

Reply:
ok. ignore it first.